builtin/trap_osh.py

```diff
@@ -262,7 +272,7 @@ def Run(self, cmd_val):
             return 1
 
         # NOTE: sig_spec isn't validated when removing handlers.
-        if code_str == '-':
+        if code_str == '-' or _IsUnsignedInteger(code_str):
```

Comment:
Can you add a link to the POSIX doc from the bug?  So readers understand why it is here

Reply:
Done